spec/support/newflow_feature_helpers.rb

```diff
@@ -118,6 +118,7 @@ def submit_signup_form
   find('#login-signup-form').click # to hide the password tooltip
   wait_for_animations
   check('signup_terms_accepted')
+  wait_for_animations
```

Comment:
does this guarantee to work?  does it slow the tests down?  stuff like this is the stuff that experienced rails devs shudder over (because as a project grows and gains thousands of tests, and when they get to taking 1 hour or so..they're often tasked to go in and remove javascript waits).  Not sure if there's a better answer or if this causes much of a delay, just bringing it up for consideration.